scripts/2500/demux-2500.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#SBATCH -c 18
+#
+#SBATCH --qos=high
+#SBATCH --time=600
+#SBATCH --mail-type=FAIL
+#SBATCH --mail-user=clinical-demux@scilifelab.se
+
+set -eu -o pipefail
+
+##########
+# PARAMS #
+##########
+
+RUN_DIR=${1?'Run dir'}
+DEMUX_DIR=${2?'Demux dir'}
+BASEMASK=${3?'Basemask'}
+UNALIGNED_DIR=${4?'Unaligned directory'}
+EMAIL=clinical-demux@scilifelab.se
+
+#############
+# FUNCTIONS #
+#############
+
+log() {
+    NOW=$(date +"%Y%m%d%H%M%S")
+    echo "[${NOW}] $*"
+
+}
+
+failed() {
+    mail -s "ERROR demultiplexing of $(basename "$RUN_DIR")" ${EMAIL} < "${DEMUX_DIR}/${UNALIGNED_DIR}/project.*.log"
+
+}
+trap failed ERR
+
+########
+# INIT #
+########
+
+log "On node: $(hostname)"
+log "starting, will use ${TMPDIR}"
+log "Run directory: ${RUN_DIR}"
+log "Demux directory: ${DEMUX_DIR}"
+
+################
+# RUN BCL2FASTQ#
+################
+
```

Comment:
You should make the /home/proj/production/flowcells/2500/"$SLURM_JOB_ID" before binding it and then clean-up after the singularity process is done.

Reply:
fixed in https://github.com/Clinical-Genomics/demultiplexing/commit/1d0004671f027e01d6cc3c1ce403a937a411fb5c